kuma/wiki/templates/wiki/includes/document_macros.html

```diff
@@ -216,7 +216,7 @@
     {% set short_share_url_gplus = share_url_gplus | bitly_shorten %}
 
     {% set twitter_url = 'http://twitter.com/share'|urlparams(url=short_share_url_twitter, text=share_text) %}
-    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=/media/redesign/img/favicon144.png&p[title]=' + share_text %}
+    {% set facebook_url = 'http://facebook.com/sharer.php?s=100&p[url]='+short_share_url_facebook+'&p[images][0]=' + MEDIA_URL + 'img/favicon144.png&p[title]=' + share_text %}
```

Comment:
Note to self: I need to look into whether a full URL (like `https://developer.cdn.mozilla.net...`) would be valid here.